src/core/jit/store.rs

```diff
@@ -1,22 +1,55 @@
+use std::collections::HashMap;
+
 use crate::core::jit::model::FieldId;
 
+/// Path to the data in the store with info
+/// to resolve nested multiple data
+#[derive(Debug, Clone)]
+pub struct DataPath {
+    /// List of paths, where every entry contains info about specific
+    /// level ot multiple data
+    multiple_path: Vec<usize>,
+}
```

Comment:
```suggestion
/// List of paths, where every entry contains info about specific
/// level ot multiple data
pub struct DataPath(Vec<usize>)
```

Can be converted to a newtype.

Reply:
replaced